R/grid_code_paths.R

```diff
@@ -22,10 +22,14 @@ tune_grid_loop <- function(resamples,
     rng
   )
 
-  resamples <- pull_metrics(resamples, results, control)
-  resamples <- pull_notes(resamples, results, control)
-  resamples <- pull_extracts(resamples, results, control)
-  resamples <- pull_predictions(resamples, results, control)
+  # carry out arranging by id before extracting each element of results (#728)
+  resample_ids <- grep("^id", names(resamples), value = TRUE)
+  id_order <- vctrs::vec_order(resamples[resample_ids])
```

Comment:
I don't suspect this is any different than passing those columns to `arrange()` (some locale technicalities maybe?), but worth looking into.

Reply:
`arrange()` uses `vec_order_radix()` under the hood. but I don't think it should matter much here